package.json

```diff
@@ -43,6 +43,7 @@
     "minilog": "3.1.0",
     "nets": "3.2.0",
     "scratch-parser": "4.3.2",
+    "scratch-sb1-converter": "github:mzgoddard/scratch-sb1-converter#to-sb2",
```

Comment:
We'll change this to a version once it's published to npm and before merging this PR.